network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -1,7 +1,6 @@
 from django.core.exceptions import ValidationError
```

Comment:
Can we have the `areas_of_focus` panel moved up to after "cause_statement"? This way it matches the section order on homepage.

Reply:
Yes, good call. Will do and commit shortly. Marking this as resolved.